src/pkcs15init/pkcs15-muscle.c

```diff
@@ -69,7 +69,7 @@ muscle_create_dir(sc_profile_t *profile, sc_pkcs15_card_t *p15card, sc_file_t *d
 	struct sc_file *file;
 	struct sc_path path;
 	memset(&file, 0, sizeof(file));
-	sc_format_path("3F00", &path);
+	sc_format_path("3F005015", &path);
```

Comment:
Is there a specific reason for changing the directory (or is it just a matter of taste)?

Reply:
Yes, at some point opensc try to access this directory (3F005015). I tried without this change and does not work :(

Reply:
Sorry, you need to dig deeper! The codes selects 3F00 and then creates 5015. This should perfectly fine...